pages/common/gcalcli.md

```diff
@@ -0,0 +1,25 @@
+# gcalcli
+
+> Command line tool to interact with Google Calendar.
+> Requests Google API authorization upon first launch.
+> More information: <https://github.com/insanum/gcalcli>.
+
+- List your events for all your calendars over the next 7 days:
+
+`gcalcli agenda`
+
+- List events from a specific one of your Google Calendars:
```

Comment:
I think we can just shorten it to - "List events from a specific calendar:"

Reply:
Hi everyone, thanks for the feedback. I think this is my first or one of my first pull requests by the way, so thanks for bearing with me. I made the changes you requested.

I left off ```gcalcli add``` earlier because its interactive prompt is much [harder to use](https://github.com/insanum/gcalcli/issues/463) as it's not well documented and each line of the prompt has a different syntax! It was added to the program after ```quick```. Anyway, I've given the basic command here as per your requests and I've added in an example specifying agenda start-stop dates.

Reply:
made this change and added an example specifying start-stop dates for showing calendar